### PR TITLE
Scrape more data

### PR DESCRIPTION
**Pull request by billfeng**

### CODE REVIEW

File: scripts/scrape-station-names.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console */
+/* eslint-disable no-plusplus */
+
+const fs = require('fs');
+const fetch = require('isomorphic-fetch');
+
+let count = 0;
+const APIKEY = '';
+const PORTION = 9;
```
Comment (yanarchy) [inline scripts/scrape-station-names.js]: What's portion?

File: scripts/scrape-station-names.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+const stations = JSON.parse(fs.readFileSync('../data/station-en.json')).slice(PORTION * 1000, (PORTION + 1) * 1000);
+
+// Using timeout (fast and does not exceed rate-limit)
+const func = async (station) => {
```
Comment (yanarchy) [inline scripts/scrape-station-names.js]: Can you give this a real name besides func?

File: scripts/scrape-station-names.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      lat: 0,
+      lng: 0,
+    };
+    console.log('\n----- Failed -----\n', station, '\n------------------\n');
+    fs.appendFile(`../data/station-cords-${PORTION}.json`, `${JSON.stringify(empty)},\n`, () => {});
+  }
+};
+
+stations.map(station =>
+  setTimeout(() => func(station), 50 * count++),
```
Comment (yanarchy) [inline scripts/scrape-station-names.js]: Why 50?

**Review by yanarchy — CHANGES_REQUESTED** [review]

**Review by yanarchy — CHANGES_REQUESTED** [review]
Please also update the README with information on these scripts and their usage.

File: scripts/scrape-station-names.js (hunk 3 above)
Comment (billfeng) [reply scripts/scrape-station-names.js]: Google's api has a 50 request per seconds limit. Setting this to 50 milliseconds means that 20 requests will be sent per second, well below Google's limit even if the rate fluctuates a little due to latency, and remains a lot faster than requesting synchronously.

File: scripts/scrape-station-names.js (hunk 2 above)
Comment (billfeng) [reply scripts/scrape-station-names.js]: Fixed!

**Review by yanarchy — APPROVED** [review]

**Review by yanarchy — DISMISSED** [review]
Please write information about the script in the README.

**Review by pegaheinaliyan — DISMISSED** [review]

**Pull request merged**